### PR TITLE
Changes required to run on v 15

### PR DESCRIPTION
**Pull request by RosSinch**

### CODE REVIEW

File: base_import_async/__manifest__.py [hunk 1]
```diff
@@ -11,8 +11,15 @@
...
+    "data": ["data/queue_job_function_data.xml"],
+    "assets": {
+        "web.assets_backend": [
+            "base_import_async/static/src/js/import.js",
+        ],
+        'web.assets_qweb': [
+            'base_import_async/static/src/xml/import.xml',
+        ],
+    },
     "installable": False,
```
Comment (guewen) [inline base_import_async/__manifest__.py:23]: Shouldn't the module be flagged as installable now?

Comment (RosSinch) [reply base_import_async/__manifest__.py:23]: It could be, but not should. (in v_14 it was False)
This PR is about necessary changes.

Comment (guewen) [reply base_import_async/__manifest__.py:23]: If it was false in 14.0, it means you could not install the module (not migrated). I may I have misunderstood the goal of your PR, do you expect the module to be installable in Odoo once merged?

**Comment by RosSinch** [comment]
@guewen 
I appreciate your patience. I saw my mistake. I've opened a new PR.
Could you please review it?
https://github.com/OCA/queue/pull/430

**Comment by RosSinch** [comment]
https://github.com/OCA/queue/pull/430

**Pull request closed without merging**